modules.d/95udev-rules/module-setup.sh

```diff
@@ -56,7 +55,11 @@ install() {
     # eudev rules
     inst_rules 80-drivers-modprobe.rules
     # legacy persistent network device name rules
-    [[ $hostonly ]] && inst_rules 70-persistent-net.rules
+    if dracut_module_included "network"; then
```

Comment:
The `network` module is a "meta-module" included along with the other network modules. Maybe this patch is only needed with the `network-legacy` module? or is there a specific network module incompatible with these rules?

Reply:
Hm, actually, I want to override 99-default.link, which gets included with all 3 network modules:

✔ ~/Entwicklung/git/github-tblume/dracut-suse [SUSE/055|…3⚑ 4] 
16:22 $ grep -r 99-default.link modules.d/
modules.d/35network-legacy/module-setup.sh:        inst_multiple -o "${systemdutildir}/network/99-default.link"
modules.d/01systemd-networkd/module-setup.sh:        "$systemdutildir"/network/99-default.link \
modules.d/35network-manager/module-setup.sh:        inst_multiple -o "${systemdutildir}/network/99-default.link"

Not sure if network-legacy would be enough for SLE, since customers might also choose to use NetworkManager, which is available in the Desktop module.

Reply:
Ok, I was misunderstanding the commit message :)

Reply:
You are right, I've updated the commit message.